src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -83,9 +94,11 @@ public void stopMotors() {
   public void periodic() {
     motorVelocity = leftShootMotor.getVelocity().getValue();
     motorsAtShootingSpeed = motorVelocity <= targetVelocity + 10 && motorVelocity >= targetVelocity - 10;
+    positionOfArm = pivotMotor.getPosition().getValue() * 360;
```

Comment:
Our position isn't based on the motor which is pre-gearbox. It's based on the wrist encoder which is mounted after the gearbox.